pydoc/usage/dataframe.rst

```diff
@@ -0,0 +1,77 @@
+***********
+DataFrames in Arkouda
+***********
+
+Like Pandas, Arkouda supports ``DataFrames``. The purpose and intended functionality remains the same in Arkouda, but are configured to be based on ``arkouda.pdarrays``.
+
+.. autoclass:: arkouda.DataFrame
+
+Data Types
+==========
+Currently, ``DataFrames`` support 4 Arkouda data types for supplying columns.
+
+* ``Arkouda.pdarray``
+* ``Arkouda.Strings``
+* ``Arkouda.Categorical``
+* ``Arkouda.SegArray``
+
+Data within the above objects can be of the types below. Please Note - Not all listed types are compatible with every type above.
+
+* ``int64``: 64-bit signed integer
```

Comment:
`uint64` also???

Reply:
@mhmerrill Yes it does support uint64. I did some additional testing this AM to verify that there are no issues with functionality when using uint64. I verified that SegArrays are also functional with uint64. Documentation has been updated to reflect.

Reply:
Cool! thanks!